.gitignore

```diff
@@ -0,0 +1,50 @@
+# build artifacts
```

Comment:
Please, don't add `.gitignore` into git at project level. Use it locally (not under version control) in your clone, if you need it.

I, for one, really want to know if there is a mess in my project tree.